src/config/config.go

```diff
@@ -13,11 +13,11 @@ type Configuration struct {
 	FileInputOption           string `json:"dataOption"`
 	StaticFile                string `json:"staticFilePath"`
 	OutlierDetectionAlgorithm string `json:"outlierDetectionAlgorithm"`
-	MadParameters             string `json:"madParameters"`
 	Prefix                    string `json:"prefix"` // can input a list of string with values seperated by a comma
 	Asn                       string `json:"asn"`
 	PeerIP                    string `json:"peerIP"`
 	Connector                 string `json:"connector"`
+	WindowSize                string `json:"windowSize"`
```

Comment:
Do we need to add 'omitempty' here for any values that were not passed in? In the same thought - what would happen if no value is passed in (there should be some default values for stuff established if you are not checking for values 😄 )